### DISCUSSION TITLE
Adding a joomla plugin short code in a form

### DISCUSSION THREAD

**Discussion by mc-cm** [joomlaboat/custom-tables#88] [Q&A] [answered]
Hello!
I need to secure a publicly available form with something like a captcha.
I installed https://kubik-rubik.de/ecc-easycalccheck-plus which should do the job. 
How can I add the {easycalccheckplus} shortcode to the form? Just putting it in a div does not display anything.

Thanks you!

**Comment by mc-cm** [comment]
of course I set "allow content plugins" to yes

**Reply by joomlaboat** [reply, marked as answer]
Custom Tables has a tag for that: {{ html.captcha(public key, secure key) }}

**Reply by mc-cm** [reply]
I guess I need a google key pair?
Is there a way to use the easycalccheck plugin. It doesn't need anything external services

**Reply by joomlaboat** [reply]
Unfortunately no, maybe I will add easycalccheck or similar in the future.

**Reply by mc-cm** [reply]
thanks!
I'll find a way around or just see if many unwanted data entries happen without captcha.

**Reply by joomlaboat** [reply]
There is also a Signature field type that will require drawing a signature (it will be saved as PNG) and this method doesn't require third-part integrations.

https://joomlaboat.com/custom-tables-wiki?document=04.-Field-Types#Signature

**Reply by mc-cm** [reply]
I'll check it out.
Good idea, actually!

**Reply by mc-cm** [reply]
sorry to keep you busy with this thread:
on my local installation it does not work. 
![image](https://github.com/joomlaboat/custom-tables/assets/1061216/3b4ad5b9-8153-423a-bfc8-617dc96d45c8)
Do I need specific libs to makle it work.
Maybe on the webserver it will work anyway?

**Comment by joomlaboat** [comment]
May I see your website?

You can contact me via email: support@joomlaboat.com

**Reply by mc-cm** [reply]
it's still local, sorry. I'll put it online tonight and will contact you it it does not work.

**Comment by joomlaboat** [comment]
Is your website in the root document directory?

**Reply by mc-cm** [reply]
no, it sits inside a subfolder below the apache htdocs folder
I run many sites on my dev machine

**Reply by joomlaboat** [reply]
I think that's the problem. I will check it on my side also.
